.travis.yml

```diff
@@ -39,5 +39,6 @@ notifications:
   webhooks:
     urls:
       - https://webhooks.gitter.im/e/c15f764f44886af1cf87
+      - https://www.travisbuddy.com/
     on_success: always
```

Comment:
IMO we should disable this for all notifications.
Created https://github.com/coala/corobo/issues/524 to get solved while this is blocked.